build.gradle~

```diff
@@ -0,0 +1,48 @@
+allprojects {
```

Comment:
This looks like it was checked in unintentionally.

It would be a good idea to add `*~` to the `.gitignore` file